Volume/Supporting Views/VolumeMessage.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  VolumeMessage.swift
+//  Volume
+//
+//  Created by Cameron Russell on 11/16/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+enum Message {
+    case noBookmarks
+    case upToDate
+    
+    var title: String {
+        switch self {
+        case .noBookmarks:
+            return "Nothing to see here!"
+        case .upToDate:
+            return "You're up to date!"
+        }
+    }
+    
+    var subtitle: String {
+        switch self {
+        case .noBookmarks:
+            return "You have no saved articles"
+        case .upToDate:
+            return "You've seen all new articles from the publications you're following."
+        }
+    }
+}
+
+struct VolumeMessage: View {
```

Comment:
Same with the newline. The enum above could probably use it too, although maybe not. Structs should have it though for sure I think.